go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/juju/usso
+
+go 1.12
```

Comment:
Make sure your CI has a recent version of 1.11 with this line. `go 1.12` in a dependency's `go.mod` caused a build failure for me with 1.11.2 yesterday in another project, but upgrading to 1.11.5 fixed that.